Cargo.toml

```diff
@@ -12,7 +12,7 @@ actix-rt = { version = "2.8.0", optional = true }
 actix-web = { version = "4", optional = true, features = ["macros"] }
 anyhow = { version = "1.0.70", optional = true }
 async-trait = { version = "0.1.68", optional = true }
-bitcoin = { version = "0.29.2", optional = true }
+bitcoin = { version = "0.30.0", optional = true }
 bitcoincore-rpc = { version = "0.16.0", optional = true }
```

Comment:
it compiles if you change in `Cargo.toml`
```toml
bitcoincore-rpc = { version = "0.16.0", git = "https://github.com/rust-bitcoin/rust-bitcoincore-rpc.git", optional = true }
bitcoincore-rpc-json = { version = "0.16.0", git = "https://github.com/rust-bitcoin/rust-bitcoincore-rpc.git", optional = true }
```

Reply:
ideally add `rev =  "201914e74b0d1fc8528093467a2daffcaeabafc3"` to each since thats the commit of rust-bitcoin/rust-bitcoincore-rpc#286